### PR TITLE
Remove tasks from admin dashboard

### PR DESCRIPTION
**Pull request by Benzene06**

### CODE REVIEW

**Comment by Kodoh** [comment]
You have some merge conflicts to resolve then I can give it a test :)

**Comment by Benzene06** [comment]
> You have some merge conflicts to resolve then I can give it a test 

Hope that sorts it :)

File: dashboard/dashboard.js [hunk 1]
```diff
@@ -20,6 +20,7 @@ function GrabProjects(){
                 let card = "<div class='card' style='width: 14rem; margin-left: 10px; margin-right: 10px;'>";
                 card += "<div class='card-body'>";
                 card += "<p class='card-title'>"+ project['projectName'] +"<p>";
+                card += "<button type='button' id='hideCard' onclick='removeTask()' class='btn btn-danger'>X</button>";
```
Comment (Kodoh) [inline dashboard/dashboard.js:23]: Should we not be replicating this in adminDashboard.php?

Comment (Benzene06) [reply dashboard/dashboard.js:23]: The button should just appear on the project after it's been created from adminDashboard.php. Doesn't seem like there's a good place to put the button on that set of code

**Pull request closed without merging**